remyxai/api/tasks.py

```diff
@@ -86,3 +86,32 @@ def train_generator(model_name: str, hf_dataset: str):
     params = {"hf_dataset": hf_dataset}
     response = requests.post(url, headers=HEADERS, params=params)
     return response.json()
+
```

Comment:
Also, move this to the top at the imports but add 
```
from typing import Optional
```
for the suggested changes to work

Reply:
Added new commit :)